modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7325,6 +7325,8 @@ feature.flag.LPS-171364.description=Users can move changes between publications.
 feature.flag.LPS-171364.title=Move Changes Between Publications
 feature.flag.LPS-174417.description=Site page and site template administrators will be able to see warnings when friendly URLs of pages already exist in the site template or the site created from the site template.
 feature.flag.LPS-174417.title=Site Template Friendly URL Conflict Warnings
+feature.flag.LPS-174455.description=Via Custom Objects OOTB API, users can add, get or update fields defined as attachment using Base64 format.
```

Comment:
I don't see the "OOTB" chain as a usual pattern in the Language.properties. Maybe I would get rid of it and could replace with `Via Custom Objects REST API, users can add....`

What do you think?

Reply:
@ccorreagg the description was prepared by PMs, so I'll consult with them, thanks!